Makefile

```diff
@@ -112,7 +112,10 @@ endif
 
 .PHONY: generate-clean
 generate-clean:
-	rm -rf hack/code-gen
+	rm -rf ./hack/code-gen
+
+generate-manifests:
```

Comment:
This target makes a lot of sense to me but why do we need to commit the manifests as part of the repo? We should have only one source of truth and that's `kudo init` for now.

Reply:
this is exactly consistent with our CRD generation

Reply:
if anything, I'd like to get rid of the `config/crds` folder altogether. we only need for tests but once we have a `beforeAll` step in kuttl, we could remove the in favor of `kudo init` call. I'd rather not add new manifests there

Reply:
I mostly agree - things that can be generated shouldn't be checked in usually.

I'm ok with having them checked in, especially the CRDs, as the controller-gen step takes quite a while. Would be nicer to not have them checked in though.